app/src/main/res/layout/fragment_about_us.xml

```diff
@@ -2,75 +2,88 @@
   ~ This project is licensed under the open source MPL V2.
   ~ See https://github.com/openMF/self-service-app/blob/master/LICENSE.md
   -->
-<androidx.coordinatorlayout.widget.CoordinatorLayout
-    xmlns:android="http://schemas.android.com/apk/res/android"
-    android:background="@drawable/bg_about"
-    android:layout_gravity="center"
-    android:layout_height="match_parent"
+<androidx.coordinatorlayout.widget.CoordinatorLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
![image](https://user-images.githubusercontent.com/70195106/103517585-6582d280-4e98-11eb-99c5-865c9db04777.png)
I think this might be a typo or build error. Please confirm it in terminal before creating pr. Thank you.